tests/platform_tests/api/test_sfp.py

```diff
@@ -538,6 +542,14 @@ def test_tx_disable_channel(self, duthosts, enum_rand_one_per_hwsku_hostname, lo
                 tx_disable_chan_mask = sfp.get_tx_disable_channel(platform_api_conn, i)
                 if self.expect(tx_disable_chan_mask is not None, "Unable to retrieve transceiver {} TX disabled channel data".format(i)):
                     self.expect(tx_disable_chan_mask == expected_mask, "Transceiver {} TX disabled channel data is incorrect".format(i))
+
+            # Restore by enabling all lanes and disabling what was disabled originally
+            sfp.tx_disable_channel(platform_api_conn, i, 0xF, False)
```

Comment:
At the beginning of the test, if we started with some channels disabled (maybe because of some breakout configuration), then at the end we are leaving all changes enabled - and thus not in the original state that the port was.